website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -67,7 +67,7 @@ The following list outlines field hierarchy, language-specific data types, and r
 <Tab heading="YAML" group="yaml">
 
 - [`apiVersion`](#apiversion): string | required | must be set to `consul.hashicorp.com/v1alpha1`
-- [`kind`](#kind): string | required | must be set to `jwtProvider`
+- [`kind`](#kind): string | required | must be set to `JWTProvider`
```

Comment:
the kind can be seen here: https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/templates/crd-jwtproviders.yaml#L19